src/nanoemoji/write_font.py

```diff
@@ -145,9 +145,21 @@ def _ufo(config: FontConfig) -> ufoLib2.Font:
     ufo.info.familyName = config.family
     # set various font metadata; see the full list of fontinfo attributes at
     # https://unifiedfontobject.org/versions/ufo3/fontinfo.plist/#generic-dimension-information
-    ufo.info.ascender = config.ascender
-    ufo.info.descender = config.descender
     ufo.info.unitsPerEm = config.upem
+    # we just use a simple scheme that makes all sets of vertical metrics the same;
+    # if one needs more fine-grained control they can fix up post build
+    ufo.info.ascender = (
+        ufo.info.openTypeHheaAscender
+    ) = ufo.info.openTypeOS2TypoAscender = config.ascender
+    ufo.info.descender = (
+        ufo.info.openTypeHheaDescender
+    ) = ufo.info.openTypeOS2TypoDescender = config.descender
+    ufo.info.openTypeHheaLineGap = ufo.info.openTypeOS2TypoLineGap = config.linegap
```

Comment:
After comparing the ttx files, I found that sTypoLineGap is set to a non-zero value here. I am not sure if there will be problems in IE 11.
https://github.com/fontello/svg2ttf/commit/0f25b4081d91c209787bdfd999b2ef574461fedc

Reply:
it is set to 0 by default. Just leave it as is if you want to keep it so

Reply:
The current behavior will set openTypeHheaLineGap to always be equal to openTypeOS2TypoLineGap, but the behavior of svg2ttf is that they are not equal. Please see this test case:

https://github.com/fontello/svg2ttf/pull/102/commits/66c503e185c7cb8786f4426fe47acdde68e70ddf#diff-a561630bb56b82342bc66697aee2ad96efddcbc9d150665abd6fb7ecb7c0ab2fR193-R199

Reply:
Well, I argue that svg2ttf is wrong, and there's nothing bad with having hhea.lineGap be equal to OS/2.sTypoLineGap (in presence of USE_MY_METRICS flag). You know, that's exactly what I meant when I said that this topic is "contentious".
I _don't_ want to get sucked into this never ending discussions, so I kindly ask you to don't insist further. These defaults are good enough for supporting modern environments and they are correct for the limited scope of emoji fonts. If you need more fine-grained control, or you need to support some legacy environments, you can write a fontTools script to fix up the values after you have built the fonts.

Reply:
I just further confirm whether there is a problem with the implementation of svg2ttf. It is indeed possible not to consider IE11 too much at present. Thank you for your PR.